runtime/dma/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+// Licensed under the Apache-2.0 license.
+
+#![cfg_attr(target_arch = "riscv32", no_std)]
+
+#[cfg(target_arch = "riscv32")]
```

Comment:
There's nothing RISC-V-specific about the HIL. It makes the build happier if we include it for all targets. (Same for mailbox.)

```suggestion
```